app/win/BUILD.gn

```diff
@@ -31,7 +31,6 @@ executable("electron_app") {
   ]
 
   sources = [
-    "//electron/atom/browser/resources/win/atom.rc",
```

Comment:
I'm actually not sure about this change, but we can delete the atom.manifest file